easybuild/easyconfigs/c/CUDA/CUDA-11.0-iccifort-2020.1.217.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'Bundle'
+name = 'CUDA'
+version = '11.0'
```

Comment:
Is there really such a benefit from dropping the last version? 
If the purpose is to treat this like we do for Java, we it would either need to be
1. CUDAcore/11.0 a wrapper at the core level that toolchains use (but I don't think we ever want to change the CUDA behind the back of a toolchain)
2. Both CUDA/11.0.2  and the "wrapper" CUDA/11.0 for all the CUDA's installed under iccifort.

There are some important nuances here, especially when it comes to HMNS

Reply:
I think it adds site-specific flexibility. 

If a site installs CUDA, but wants to install a revision that is more recent than what EasyBuild upstream provides for, that site can still use all of the EasyConfig which use `('CUDA', '11.0')` as dependencies without having to change a lot of easyconfigs

Reply:
OK, hm, well, I'm not saying I'm against it but just some thoughts:
1. This is just the CUDA part so, you're still stuck when it comes to the compiler and such.
2. Wouldn't be necessary for the fosscuda level, so that limits the amount of software affected anyway
3. It would come back to bite us (slightly) if we ever decided to also add `CUDA-11.0.3-iccifort-2020.1.217`
Both the for- and against arguments seem to be somewhat theoretical though, so I'm not sure it matters much.

Just to be clear, as it stands, this is very different implementation (with a different motivation/rationale) from what @boegel mentioned in the chat:
> We do this for Java too. And we should consider doing it for more stuff, like Python, and do in-place updates.

This can not do in-place updates, and, I'm not sure it ever can, considering it's part of a toolchain hierarchy thing.

Reply:
The idea is that the toolchains also use `('CUDA', '11.0')`. See the `gcccuda-2020a.eb` and `iccifortcuda-2020a.eb` attached to this PR. 

A site who wants to install `CUDAcore-11.0.3.eb` and use that as their base only has to change the two files`CUDA-11.0-iccifort-2020.1.217.eb` and `CUDA-11.0-GCC-9.3.0.eb` files, since toolchains depend on those, and not directly on `CUDAcore-11.0.2.eb`

Reply:
Maybe @bartoldeman and @boegel can pitch in on this

Reply:
There was some discussion in the conf call (I wasn't there, it's the early one): https://github.com/easybuilders/easybuild/wiki/Conference-call-notes-20200708

I'd stay with 11.0.3, it makes more sense if you look at the other toolchains (from an upstream perspective, from a CC perspective 11.0 is ok but that also has to do in how we expose cuda to users).

Reply:
ok. I restored the revision number.